docs/source/install.rst

```diff
@@ -33,107 +24,81 @@ Cap has a required dependency on C++14.
+Please refer to the deal.II documentation to see how to install `p4est
+<https://dealii.org/developer/external-libs/p4est.html>`_ and `Trilinos
+<https://dealii.org/developer/external-libs/trilinos.html>`_.
 
-The open source finite element library deal.II is optional.
-It is only required to work with energy storage devices of type ``SuperCapacitor``.
-Version 8.4.0 or later compiled with C++14/MPI/Boost/p4est/Trilinos support is required.
-The development sources can be found `here <https://github.com/dealii/dealii>`_.
-
-To download the release version 8.4.0, do:
-
-.. code::
-
-    $ wget --output-document=${PREFIX}/archive/dealii-8.4.0.tar.gz \
-        https://github.com/dealii/dealii/releases/download/v8.4.0/dealii-8.4.0.tar.gz
-    $ mkdir ${PREFIX}/source/dealii && tar -xf ${PREFIX}/archive/dealii-8.4.0.tar.gz \
-        -C ${PREFIX}/source/dealii --strip-components=1
-
-It is a good idea to make a `configure_dealii` script such as:
-
-.. code-block:: bash
-    :linenos:
-
-    EXTRA_ARGS=$@
-
-    cmake                                                \
-        -D CMAKE_INSTALL_PREFIX=${PREFIX}/install/dealii \
-        -D CMAKE_BUILD_TYPE=Release                      \
-        -D DEAL_II_WITH_CXX14=ON                         \
-        -D DEAL_II_WITH_MPI=ON                           \
-        -D BOOST_DIR=${PREFIX}/install/boost             \
-        -D P4EST_DIR=${PREFIX}/install/p4est             \
-        -D TRILINOS_DIR=${PREFIX}/install/trilinos       \
-        $EXTRA_ARGS                                      \ 
-        ${PREFIX}/source/dealii
-
-Then run:
-
-.. code::
 
-    $ mkdir ${PREFIX}/build/dealii && cd ${PREFIX}/build/dealii
-    $ ../configure_dealii
-    $ make -j<N> install
-
-Please refer to the deal.II documentation to see how to install
-`p4est <https://dealii.org/developer/external-libs/p4est.html>`_ and
-`Trilinos <https://dealii.org/developer/external-libs/trilinos.html>`_.
-
-Install cap from source
+Install Cap from source
 -----------------------
 Get the source:
 
 .. code::
 
-    $ git clone https://github.com/ORNL-CEES/cap.git ${PREFIX}/source/cap
+    $ git clone https://github.com/ORNL-CEES/Cap.git && cd Cap
 
-Create a `configure_cap` script in `${PREFIX}/build`:
+Create a ``configure_cap.sh`` script such as:
 
 .. code-block:: bash
     :linenos:
 
+    #!/usr/bin/env bash
+
     EXTRA_ARGS=$@
 
     cmake \
-        -D CMAKE_INSTALL_PREFIX=${PREFIX}/install/cap \
-        -D BOOST_DIR=${PREFIX}/install/boost \
+        -D CMAKE_INSTALL_PREFIX=<your/install/prefix/here> \
+        -D BOOST_DIR=<path/to/boost> \
         -D ENABLE_DEAL_II=ON \
-        -D DEAL_II_DIR=${PREFIX}/install/dealii \
+        -D DEAL_II_DIR=<path/to/dealii> \
         $EXTRA_ARGS \ 
-        ${PREFIX}/source/cap
+        ..
 
 Configure, build and install:
 
 .. code::
 
-    $ mkdir ${PREFIX}/build/cap
-    $ cd ${PREFIX}/build/cap
-    $ ../configure_cap
+    $ mkdir build && cd build
+    $ vi configure_cap.sh
```

Comment:
I literally wrote "create a script" above. You want me to add a comment on that line?
```
$ [edit the file with your favorite text editor]
```